src/pages/Portfolio/ImpactDashboard/LendingInsights.vue

```diff
@@ -283,21 +415,58 @@ export default {
 										totalCount
 									}
 								}
+								loanStatsByYear {
+									amount
+									count
+								}
+								countriesLentToByYear
 							}
 							userStats {
 								amount_of_loans
 								number_of_loans
 							}
 						}
+						lend {
+							percentilePerYear {
```

Comment:
This needs to be provided an amount to get the percentile data for the user. It is the total amount the user has lent from `loanStatsByYear`. So this might need to be split into two calls?